airbyte-integrations/connectors/source-hubspot/source_hubspot/streams.py

```diff
@@ -259,6 +260,11 @@ def __init__(self, api: API, start_date: Union[str, pendulum.datetime], credenti
         elif creds_title in (OAUTH_CREDENTIALS, PRIVATE_APP_CREDENTIALS):
             self._authenticator = api.get_authenticator()
 
+    def should_retry(self, response: requests.Response) -> bool:
+        if response.status_code == HTTPStatus.FORBIDDEN:
+            setattr(self, "raise_on_http_errors", False)
```

Comment:
are we logging tot he user when this happens?

Reply:
updated